### PR TITLE
Corrected validations in Converters

### PR DESCRIPTION
**Pull request by tomasprochazka2**
Fixed wrong validation condition in noexcept qualifier. Also corrected condition in convert call when validate==true.

### CODE REVIEW

File: include/superior_mysqlpp/converters/to_integer.hpp [hunk 1]
```diff
@@ -24,13 +24,13 @@ namespace SuperiorMySqlpp { namespace Converters
...
             static_assert(std::numeric_limits<T>::digits10, "length > digits10");
-            static inline void call(T& result, const char*& str) noexcept(validate)
+            static inline void call(T& result, const char*& str) noexcept(!validate)
             {
                 static constexpr T base = pow(10UL, length-1);
                 char c = *str;
                 if (validate)
                 {
-                    if (c < '0' || c > '9')
+                    if (c!>= '0' && c!<= '9') //This is sometimes faster then using ||
```
Comment (MrWarlockX) [inline include/superior_mysqlpp/converters/to_integer.hpp]: You have syntax error on this line (that exclamation marks)

File: include/superior_mysqlpp/converters/to_integer.hpp [hunk 2]
```diff
@@ -45,12 +45,12 @@ namespace SuperiorMySqlpp { namespace Converters
...
         {
             static_assert(std::numeric_limits<T>::digits10, "length > digits10");
-            static inline void call(T& result, const char*& str) noexcept(validate)
+            static inline void call(T& result, const char*& str) noexcept(!validate)
             {
                 char c = *str;
                 if (validate)
                 {
-                    if (c < '0' || c > '9')
+                    if (c!>= '0' && c!<= '9') //This is sometimes faster then using ||
```
Comment (MrWarlockX) [inline include/superior_mysqlpp/converters/to_integer.hpp]: Same thing over here (syntax error)

File: include/superior_mysqlpp/converters/to_integer.hpp [hunk 3]
```diff
@@ -45,12 +45,12 @@ namespace SuperiorMySqlpp { namespace Converters
...
         {
             static_assert(std::numeric_limits<T>::digits10, "length > digits10");
-            static inline void call(T& result, const char*& str) noexcept(validate)
+            static inline void call(T& result, const char*& str) noexcept(!validate)
             {
                 char c = *str;
                 if (validate)
                 {
-                    if (c < '0' || c > '9')
+                    if (!(c>= '0' && c<= '9')) //This is sometimes faster then using ||
```
Comment (MrWarlockX) [inline include/superior_mysqlpp/converters/to_integer.hpp]: Please add spaces after `c` variable and after beginning of the comment.

File: include/superior_mysqlpp/converters/to_integer.hpp [hunk 4]
```diff
@@ -24,13 +24,13 @@ namespace SuperiorMySqlpp { namespace Converters
...
             static_assert(std::numeric_limits<T>::digits10, "length > digits10");
-            static inline void call(T& result, const char*& str) noexcept(validate)
+            static inline void call(T& result, const char*& str) noexcept(!validate)
             {
                 static constexpr T base = pow(10UL, length-1);
                 char c = *str;
                 if (validate)
                 {
-                    if (c < '0' || c > '9')
+                    if (!(c >= '0' && c <= '9')) //This is sometimes faster then using ||
```
Comment (MrWarlockX) [inline include/superior_mysqlpp/converters/to_integer.hpp]: Please add space after beginning of the comment.

Comment (tomasprochazka2) [reply include/superior_mysqlpp/converters/to_integer.hpp]: ok

File: include/superior_mysqlpp/converters/to_integer.hpp (hunk 3 above)
Comment (tomasprochazka2) [reply include/superior_mysqlpp/converters/to_integer.hpp]: ok

**Review by smradCZ — DISMISSED** [review]

**Review by smradCZ — APPROVED** [review]

**Pull request merged**